source/widgets/UnixWindow.ooc

```diff
@@ -22,26 +22,51 @@ use ooc-draw
 use ooc-opengl
 use ooc-x11
 
-version((unix || apple) && !gpuOff) {
-UnixWindow: class extends DisplayWindow {
-	_xWindow: X11Window
-	_openGLWindow: OpenGLWindow
-	context ::= this _openGLWindow context
-	init: func (size: IntSize2D, title: String) {
-		super()
-		this _xWindow = X11Window new(size, title)
-		this _openGLWindow = OpenGLWindow new(this _xWindow)
+version(unix || apple) {
+	UnixWindowBase: class extends DisplayWindow {
```

Comment:
It has no abstract methods. In fact `UnixWindow` in `nogpu` mode should be a typedef for this one. But since ooc does not support typedefs...